src/Nox.Types/Types/NoxUri/NoxUri.cs

```diff
@@ -0,0 +1,4 @@
+namespace Nox.Types;
+public sealed class NoxUri : ValueObject<System.Uri, NoxUri> 
```

Comment:
Uri keeps the URI and the UriKind, so this will be a Compound Type

Reply:
System.Uri already has this as a constructor. 
I believe current implementation meets our needs.